src/common/modals/ShowScreenShot.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Resolve error: unable to load resolver "alias".](https://app.codacy.com/manual/gorilla-devs/GDLauncher/pullRequest?prid=5082094)